File: 프로그래머스/lv1/86491. 최소직사각형/최소직사각형.java
class Solution {
    public int solution(int[][] sizes) {
        int x=0, y=0;
        int xmax=0, ymax=0;
        for(int i=0; i<sizes.length; i++){
            x = Math.max(sizes[i][0], sizes[i][1]);
            y = Math.min(sizes[i][0], sizes[i][1]);
            xmax = Math.max(xmax, x);
            ymax = Math.max(ymax, y);
        }
        return xmax*ymax;
    }
}

File: 프로그래머스/lv1/86491. 최소직사각형/최소직사각형.js
function solution(sizes) {
    var answer = 0;
    var maxRow = 0;
    var maxCol = 0;
    sizes.forEach(([w, h]) => {
        col = Math.min(w, h);
        row = Math.max(w, h);
        maxRow = Math.max(maxRow, row);
        maxCol = Math.max(maxCol, col);
    })
    return maxRow * maxCol;
}